### PR TITLE
doc: ClassPropertyDescriptor example

### PR DESCRIPTION
**Pull request by ross-weir**
Modified the `object_wrap` example to use `InstanceAccessor` instead of `InstanceMethod`.

### CODE REVIEW

File: doc/class_property_descriptor.md [hunk 1]
```diff
@@ -6,6 +6,88 @@ specific to each `Napi::ObjectWrap<T>` subclass.
 This prevents using descriptors from a different class when defining a new class
 (preventing the callbacks from having incorrect `this` pointers).
 
+## Example
```
Comment (gabrielschulhof) [inline doc/class_property_descriptor.md]: Nit: @ross-weir would you mind fixing the typo a few lines above this line? It reads "# Class propertry and descriptor" whereas it should read "#Class property and descriptor".

Comment (ross-weir) [reply doc/class_property_descriptor.md]: Sure thing. Fixed

File: doc/class_property_descriptor.md [hunk 2]
```diff
@@ -1,11 +1,93 @@
...
+    return this->GetValue(info);
+}
+
+// Initialize native add-on
+Napi::Object Init (Napi::Env env, Napi::Object exports) {
+    Example::Init(env, exports);
+    return exports;
+}
+
+// Regisgter and initialize native add-on
```
Comment (NickNaso) [inline doc/class_property_descriptor.md]: Tupo here  **Regisgter**  should be **Register**

Comment (ross-weir) [reply doc/class_property_descriptor.md]: Fixed here and where I copied it from :-)

**Comment by gabrielschulhof** [comment]
Landed in cab3b1e2a2264a985382531a8523fb46c60fda70.

**Pull request closed without merging**